fanficfare/adapters/adapter_readonlymindcom.py

```diff
@@ -1,6 +1,5 @@
 # -*- coding: utf-8 -*-
-# -- coding: utf-8 --
-# Copyright 2013 Fanficdownloader team, 2021 FanFicFare team
+# Copyright 2021 FanFicFare team
```

Comment:
Nittest of nit picks--it's 2022 now. :-)

Reply:
:D 

January, you know.